flink-runtime/src/main/java/org/apache/flink/runtime/query/QueryableStateUtils.java

```diff
@@ -132,9 +134,7 @@ public static KvStateServer createKvStateServer(
 					KvStateRequestStats.class);
 			return constructor.newInstance(address, ports, eventLoopThreads, queryThreads, kvStateRegistry, stats);
 		} catch (ClassNotFoundException e) {
-			final String msg = "Could not load Queryable State Server. " +
-				"Probable reason: flink-queryable-state-runtime is not in the classpath. " +
-				"Please put the corresponding jar from the opt to the lib folder.";
+			final String msg = "Could not load Queryable State Server. " + ERROR_MESSAGE_ON_LOAD_FAILURE;
 			if (LOG.isDebugEnabled()) {
 				LOG.debug(msg, e);
```

Comment:
The `debug` message should change as well as above, right?

Reply:
ah, of course...